src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportOrchestratorJob.cs

```diff
@@ -412,7 +412,7 @@ private async Task<IList<long>> EnqueueProcessingJobsAsync(IEnumerable<InputReso
             {
                 const string message = "Duplicate file detected in list of files to import.";
                 _logger.LogJobError(ex, orchestratorInfo, message);
-                throw new JobExecutionException(message, ex);
+                throw new ImportProcessingException(message, ex);
```

Comment:
> throw new ImportProcessingException(message, ex); [](http://example.com/codeflow?start=15&length=50)

What is the reason to register $import? Why not to return bad request from import controller before registration?